src/assets/styles/main.scss

```diff
@@ -55,3 +55,11 @@ text {
     display: none;
   }
 }
+
+
+.signal-grid-container {
+  display: grid;
+  gap: 1.5rem;
+  grid-template-columns: repeat(auto-fit, minmax(min(15rem, 100%), 1fr));
```

Comment:
we might want to change the `15rem` to more like 20 - 30 - this gets pretty squishy before it flips vertical

<img width="542" alt="image" src="https://user-images.githubusercontent.com/24885580/166509177-0a06d76e-58c0-439c-98bc-f5fe3f7b7147.png">

Reply:
updated to 20rem